tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -124,9 +110,6 @@ def _test_seq_snapshots(context):
         path = os.path.join(microvm.jailer.chroot_path(), VSOCK_UDS_PATH)
         check_host_connections(microvm, path, blob_path, blob_hash)
 
-        # Start a new instance of fio on each iteration.
-        _guest_run_fio_iteration(ssh_connection, i)
```

Comment:
should we completely remove it? I guess the point of this being here is to test block functionality after restoring from a snapshot. Could we not just make sure the fio process is finished before getting the next snapshot?

Reply:
@dianpopa any comments on this?

Reply:
The reason I deleted it completely is because I see this as a superfluous/wrong check; given that FIO is run in the background and we do not check for its status, if the block device malfunctions we would not receive a clear error for it; it would manifest (probably) in hangs;

Also, my assumption is that this test was introduced to check for file corruptions (that is why is run on the root device).

Now, the test in its current form runs for 10s; I do not think is a good idea to run it for that long;
Should be enough to just read the root device file once, right?